crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -227,68 +319,29 @@ public override Scope.Extensions ToExtensions()
                     // "system" | "user"
                     TableType: "user",
                     // "public" | "private"
-                    TableAccess: "{{(IsPublic ? "public" : "private")}}",
-                    Scheduled: {{(Scheduled is not null ? $"nameof({Scheduled})" : "null")}}
+                    TableAccess: "{{(v.IsPublic ? "public" : "private")}}",
+                    Scheduled: {{(v.Scheduled is not null ? $"nameof({v.Scheduled})" : "null")}}
                 ),
                 (uint) ((SpacetimeDB.BSATN.AlgebraicType.Ref) new BSATN().GetAlgebraicType(registrar)).Ref_
-            );
+            ),
+            """))}}
+            ];
 
             static SpacetimeDB.Internal.Filter {{iTable}}.CreateFilter() => new([
                 {{string.Join(",\n", Members.Select(f => f.GenerateFilterEntry()))}}
             ]);
-
-            public static IEnumerable<{{ShortName}}> Iter() => {{iTable}}.Iter();
-            public static IEnumerable<{{ShortName}}> Query(System.Linq.Expressions.Expression<Func<{{ShortName}}, bool>> predicate) => {{iTable}}.Query(predicate);
-            public void Insert() => {{iTable}}.Insert(this);
             """
         );
 
-        foreach (
-            var (f, i) in Members
-                .Select((field, i) => (field, i))
-                .Where(pair => pair.field.IsEquatable)
-        )
-        {
-            var colEqWhere = $"{iTable}.ColEq.Where({i}, {f.Name}, BSATN.{f.Name})";
-
-            extensions.Contents.Append(
-                $"""
-                public static IEnumerable<{ShortName}> FilterBy{f.Name}({f.Type} {f.Name}) =>
-                    {colEqWhere}.Iter();
-                """
-            );
-
-            if (f.Attrs.HasFlag(ColumnAttrs.Unique))
-            {
-                extensions.Contents.Append(
-                    $"""
-                    public static {ShortName}? FindBy{f.Name}({f.Type} {f.Name}) =>
-                        FilterBy{f.Name}({f.Name})
-                        .Cast<{ShortName}?>()
-                        .SingleOrDefault();
-
-                    public static bool DeleteBy{f.Name}({f.Type} {f.Name}) =>
-                        {colEqWhere}.Delete();
-
-                    public static bool UpdateBy{f.Name}({f.Type} {f.Name}, {ShortName} @this) =>
-                        {colEqWhere}.Update(@this);
-                    """
-                );
-            }
-        }
-
         return extensions;
     }
 }
 
 record ReducerParamDeclaration : MemberDeclaration
```

Comment:
Since it doesn't have any own data, I'd probably just remove this class and store `MemberDeclaration`. You'd only need to change `ConvertMember`.

Reply:
Hrm, isn't this out of scope for this PR? Happy to do it in a future one

Reply:
Well you did remove `IsContextArg` in this PR, which makes this type 1:1 same as `MemberDeclaration`, so might as well remove the entire class.

Reply:
Good point